py/janitor/runner.py

```diff
@@ -3034,7 +3034,7 @@ async def main_async(argv=None):
     parser.add_argument(
         "--public-vcs-location",
         type=str,
-        default=None,
```

Comment:
What is this meant to do? Perhaps just set required=True for now

Reply:
Having it to 'none' wasn't happy - having it an empty string was passing.

I did think about making it a required, but as it wasn't needed before the rust conversion, didnt' want to it too much.

Reply:
It wasn't really working before the rust conversion either, because the default behaviour (which was accidental) would mean returning local vcs managers.

Reply:
In that case, updated  =)